docs/sources/explore/_index.md

```diff
@@ -59,6 +59,46 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 To close the newly created query, click on the Close Split button.
 
+## Share Explore URLs
+
+When using Explore, the URL in the browser address bar is updated as you make changes to the queries. This allows you to share the URLs with others or bookmark them for later use.
+
+### Generating Explore URLs from external tools
+
+Explore URLs have a defined structure, you can build an URL from external tools and open it in Grafana. The URL structure is:
+
+```
+http://<grafana_url>/explore?panes=<panes>&schemaVersion=<schema_version>&orgId=<org_id>
+```
+
+where:
+
+- `org_id` is the organization ID
+- `schema_version` is the schema version, currently only `1` is supported
```

Comment:
If we go with mentioning compact URLs below, we should clarify this a little. 

"`schema_version` is the schema version. Currently, we migrate URLs to version `1`"

Reply:
similarly to https://github.com/grafana/grafana/pull/71724#discussion_r1267924296, i think the objective here is to describe how to generate one for the current version without providing too many details about the whys.

probably however ``currently only `1` is supported`` is a bit confusing as it implies that there will be other schema versions with the same structure. maybe we should rephrase this to something like:

```suggestion
- `schema_version` is the schema version of the URL format, to make Explore parse the URL using the current schema it must be set to `1`
```

maybe? wdyt? (cc. @ifrost @harisrozajac )

Reply:
> - `schema_version` is the schema version of the URL format, to make Explore parse the URL using the current schema it must be set to `1` 

@Elfo404 I think this is a good description. Since it implies that there are other schema versions, would it make sense to specify somewhere what those are?

Reply:
Maybe?

```suggestion
- `schema_version` is the schema version (should be set to the latest version which is `1`)
```